### DISCUSSION TITLE
Link UNC Folder/File path?

### DISCUSSION THREAD

**Discussion by JordanSands33** [requarks/wiki#3423] [Help / Questions]
I just had a quick question. I am setting up a Wiki JS for some end users to access for training purposes. What I need to know is if there is a way to link a UNC document through the page editor? For example: The manager wants the user to be linked directly to a PDF containing some information. The path is \\TEST\info\help\help.pdf. Would I be able to directly link that in the Wiki to then have it launch that pdf on the users machine?

I did try using the link function in the Visual Editor, as well as the file:///// function in the HTML editor. Any help is appreciated! Thanks. 

**Host Info (please complete the following information):**
OS: [e.g. Ubuntu 18.04]
Wiki.js version: [2.4.101]
Database engine: [postgres 9.7]

**Comment by baz2a** [comment]
Hello,

WikiJS 2.5.144 user here...found that in the visual editor using the UNC path of your network share (no need for file:/// or anything else...using UNC here) document for your link works ok but in Chrome you have to install an 'enable local file links' extension like https://chrome.google.com/webstore/detail/enable-local-file-links/nikfmfgobenbhmocjaaboihbeocackld

Reference - https://support.google.com/gsa/answer/2664790?hl=en

After this extension was installed and configured with Site Access 'on specific sites' and have my site entered as http://www.blahblah.local/*

Once closed and reopened page, I was able to open files stored on file server.

**Comment by cancodr** [comment]
I was facing the same issue.  The Editor seems to strip the "file:///" prefix, and on top of that, browsers disable this ability due to security (you can't link to a local file from an non-local site), just as @baz2a mentioned.  Installing plugins is a bit of a nuisance, and may not work permanently.

In our case, Wiki.js was installed on an internal NAS, and I wanted the ability to link to files which are also on the same NAS.  My solution (albeit not fantastic), was to create a virtual web host with the root directory of the NAS's shared folder, so that I can create links to the NAS itself, and have the virtual host serve the files off of the local NAS filesystem.

I created a virualhost, "http://nas-server:99", that has a root directory of our shared folder.  

Eg:  Our share is mapped under Windows as "Z:\" drive.  If we have a file under Z:\Info\PDFs\file.pdf, I can create a link:

&lt;a href="http://nas-server:99/Info/PDFs/file.pdf" &gt; Link &lt;/a&gt;
